RecastDemo/Source/main.cpp

```diff
@@ -686,15 +686,16 @@ int main(int /*argc*/, char** /*argv*/)
 				mouseOverMenu = true;
 			
 			int levelToLoad = -1;
-			for (int i = 0; i < files.size; ++i)
-			{
-				if (imguiItem(files.files[i]))
+			
+			for (unsigned int i = 0; i < files.size(); ++i) {
```

Comment:
If I'm being picky, this unsigned int (and the one on line 768) should also be std::size_t to avoid warnings on 64 bit platforms, but that'll trigger all over the place already so don't worry!